<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageEx01</class>
 <widget class="QWidget" name="StageEx01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtInput"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="txtShow"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnSend">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
